.packit.yaml

```diff
@@ -8,6 +8,7 @@ synced_files:
 # packit was already taken on PyPI
 upstream_package_name: packitos
 downstream_package_name: packit
+upstream_project_url: "https://github.com/packit-service/packit"
```

Comment:
```suggestion
upstream_project_url: https://github.com/packit-service/packit
```

Reply:
Is it not needed or it breaks something?

Reply:
just not needed